modules/liveIntentIdSystem.js

```diff
@@ -64,18 +82,30 @@ function initializeLiveConnect(configParams) {
   if (configParams.partner) {
     identityResolutionConfig.source = configParams.partner
   }
+  if (configParams.ajaxTimeout) {
+    identityResolutionConfig.ajaxTimeout = configParams.ajaxTimeout;
+  }
 
   const liveConnectConfig = parseLiveIntentCollectorConfig(configParams.liCollectConfig);
   liveConnectConfig.wrapperName = 'prebid';
   liveConnectConfig.identityResolutionConfig = identityResolutionConfig;
   liveConnectConfig.identifiersToResolve = configParams.identifiersToResolve || [];
+  if (configParams.emailHash) {
+    liveConnectConfig.eventSource = { hash: configParams.emailHash }
+  }
   const usPrivacyString = uspDataHandler.getConsentData();
   if (usPrivacyString) {
     liveConnectConfig.usPrivacyString = usPrivacyString;
   }
+  const gdprConsent = gdprDataHandler.getConsentData()
+  if (gdprConsent) {
+    liveConnectConfig.gdprApplies = gdprConsent.gdprApplies ? 1 : 0;
```

Comment:
Shouldn't these be boolean?

Reply:
it as to be boolean. I'll update the pr.

Reply:
and the tests work because 1 is the same as "true" in js.

Reply:
updated https://github.com/LiveIntent/Prebid.js/pull/3/commits/c7b60e999279bc7d537379c68466826b85ca293e